Extension/src/LanguageServer/client.ts

```diff
@@ -937,7 +942,14 @@ export class DefaultClient implements Client {
             if (isCompiler) {
                 const path: string | undefined = paths[i].replace(compilerName, "");
                 const description: string = localize("found.string", "Found at {0}", path);
-                items.push({ label: compilerName, description: description, index: i });
+                const label: string = localize("use.compiler", "Use the configuration from querying {0}", compilerName);
+                items.push({ label: label, description: description, index: i });
+            } else if (paths[i] === "configuration providers") {
+                items.push({ label: localize("configuration.providers", "configuration providers"), index: i, kind: vscode.QuickPickItemKind.Separator });
+            } else if (paths[i] === "compile commands") {
+                items.push({ label: localize("compile.commands", "compile commands"), index: i, kind: vscode.QuickPickItemKind.Separator });
```

Comment:
I was thinking we keep this as `compile_commands.json` (unlocalized).  I'm not sure how "compile commands" translates and whether foreign language users would make that association since the file name is untranslated English.

Reply:
sure